lib/Raven/Configuration.php

```diff
@@ -0,0 +1,446 @@
+<?php
+
+/*
+ * This file is part of Raven.
+ *
+ * (c) Sentry Team
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Raven;
+
+use Composer\CaBundle\CaBundle;
+use Symfony\Component\OptionsResolver\Options;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * Configuration container for the Sentry client.
+ *
+ * @author Stefano Arlandini <sarlandini@alice.it>
+ */
+class Configuration
```

Comment:
> we don't need to have a monolithic class

And it\'s bad because...

> Processors require...

And they have access.

> It's common in other libraries

This is a logging library not a nuclear station\'s center.